pym/portage/package/ebuild/_parallel_manifest/ManifestScheduler.py

```diff
@@ -91,3 +84,90 @@ def _task_exit(self, task):
 					noiselevel=-1)
 
 		AsyncScheduler._task_exit(self, task)
+
+
+def _future_fetchlist(loop, portdb, repo_config, cp, cpv_list,
+	max_jobs=None, max_load=None):
+	"""
+	Asynchronous form of FetchlistDict, with max_jobs and max_load
+	parameters in order to control async_aux_get concurrency.
```

Comment:
I'd like to split out a utility function that's like [asyncio.gather](https://docs.python.org/3/library/asyncio-task.html#asyncio.gather) with `max_jobs` and `max_load` parameters.

Reply:
I've split out an `iter_gather` function.